Resources/Textures/_Nuclear14/Decals/FullWhite.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": null,
```

Comment:
Not sure if null is supported or not. You can do CC-0 if you don't care about licensing or CC-BY-SA-3.0 if you want to be credited for it properly and it to be share able.